README.md

```diff
@@ -136,267 +132,43 @@ For an even more in-depth demo, check out our [Deploying to Kubernetes](https://
-keychains into this provider in a future release._
-
-Example:
-```
-...
-    credentials:
-      rsa:
-        from: keychain
-        get: servicename#accountname
-...
-```
-
-## Health Checks
-
-Secretless broker exposes two endpoints that can be used for readiness and liveliness checks:
-
-- `http://<host>:5335/ready` which will indicate if the broker has loaded a valid configuration.
-- `http://<host>:5335/live` which will indicate if the broker has service connectors activated.
-
-If there are failures, the service will return a `503` status or a `200` if the service indicates that
-the broker is ready/live.
-
-Note: If Secretless is not provided with a configuration (e.g. it is not listening to anything),
-the live endpoint will also return 503.
-
-You can manually check the status with these endpoints by using `curl`:
-```
-$ # Start Secretless Broker in another terminal on the same machine
-
-$ curl -i localhost:5335/ready
-HTTP/1.1 200 OK
-Content-Type: application/json; charset=utf-8
-Date: Thu, 27 Dec 2018 17:12:07 GMT
-Content-Length: 3
-
-{}
-```
-
-If you would like to retrieve the full informational JSON that includes details on
-which checks failed and with what error, you can add the `?full=1` query parameter
-to the end of either of the available endpoints:
-```
-$ # Start Secretless Broker in another terminal on the same machine
-
-$ curl -i localhost:5335/ready?full=1
-HTTP/1.1 200 OK
-Content-Type: application/json; charset=utf-8
-Date: Thu, 27 Dec 2018 17:13:22 GMT
-Content-Length: 45
-
-{
-    "listening": "OK",
-    "ready": "OK"
-}
-```
+## About our releases
+
+### Docker images
+The primary source of Secretless Broker releases is our [DockerHub](https://hub.docker.com/r/cyberark/secretless-broker/tags/).
+
+Every new GitHub tag (`x.y.z`) added to the project produces a set of Docker images in DockerHub: (`x.y.z`, `x.y`, `x`, `latest`). The `latest` image tag therefore always corresponds to the latest GitHub tag.
+
+### GitHub releases
+Post-1.0, GitHub releases are created for GitHub tagged versions that have undergone additional quality activities to ensure that Secretless continues to meet its [stability requirements](#stable-release-definition). When a GitHub release is created for a stable Secretless version, an additional Docker image is pushed to DockerHub with the `stable` tag.
+
+### GitHub repository
+The code on `master` in the project's GitHub repository represents the development work done since the previous GitHub tag. It is possible to build Secretless from source (see [our contributing guidelines](#development) for more info), but for regular use we recommend using the `stable` Docker image from DockerHub or an official GitHub release.
+
+### Stable release definition
+Stable components (eg service connectors, credential providers, etc.) of Secretless meet the core acceptance criteria:
+- Synchronous user actions must complete in a reasonable amount of time
```

Comment:
What are the synchronous user actions here? We do most of our work on startup so I'm a bit confused as to what's meant here.

Reply:
maybe @andytinkham can comment / give some context here? this is pulled straight from the core acceptance criteria that he drafted.

Reply:
The "synchronous" word specifically, I think was added at @alexkalish's suggestion because we may not be able to control the timing of anything happening asynchronously. In general, the intent of this core criteria right now is "think about performance and don't make things be slow". As we discuss and refine, this should become something more measurable and less hand-wavey. For this page, I'd be OK dropping 'synchronous' though I don't know if there's anything to do yet around "reasonable".

Reply:
This AC feels a little out of place in the list, especially since this product doesn't have any synchronous user actions, in and of itself.  I think that the 20% max performance penalty added farther below captures the spirit of this criteria.

Reply:
ok, so it sounds like y'all would be good with me dropping this one?

Reply:
yup - seems like it

Reply:
I'm ok with removing it.